### PR TITLE
MGMT-13334: Failed to load feature support levels preview

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-13334

When `feature-support-levels` API isn't available or doesn't include the cluster's Openshift version, the installation page shows:

**Before:**
image

**After:**
image

### CODE REVIEW

File: src/common/components/ErrorHandling/WithErrorBoundary.tsx [hunk 1]
```diff
@@ -4,26 +4,33 @@ import { ErrorBoundary, FallbackProps } from 'react-error-boundary';
...
         variant={ButtonVariant.link}
         isInline
         key="reset-error-boundary"
       >
         try again
-      </Button>,
-    ];
+      </Button>
+    ) : (
+      <></>
```
Comment (jkilzi) [inline src/common/components/ErrorHandling/WithErrorBoundary.tsx:23]: If it is supposed to render nothing...
```suggestion
      null
```

File: src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx [hunk 2]
```diff
@@ -179,7 +179,13 @@ const SupportLevel = ({ cluster }: SupportLevelProps) => {
 };
 
 const ClusterFeatureSupportLevelsDetailItem = ({ ...props }: SupportLevelProps) => (
-  <WithErrorBoundary title="Failed to load feature support levels review">
+  <WithErrorBoundary
+    title="Feature support levels not available"
+    content={`OpenShift version ${
+      props.cluster.openshiftVersion || ''
+    } does not currently review feature support levels.`}
```
Comment (jkilzi) [inline src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx:186]: we need microcopy help here

**Review by jkilzi — CHANGES_REQUESTED** [review]
We might still need some changes

File: src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx (hunk 2 above)
Comment (jgyselov) [reply src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx:186]: What microcopy are we missing?

Comment (jkilzi) [reply src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx:186]: We could generate this message: "OpenShift version does not currently review feature support levels".
Sounds a bit weird...

Comment (jgyselov) [reply src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx:186]: Type-wise yes, we could. But in reality I don't think we'll ever have a cluster without an Openshift version on the installation page. So the  `|| ''` part is really there only to make the linters happy.

Comment (jkilzi) [reply src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx:186]: ack

**Review by jkilzi — DISMISSED** [review]

**Review by ammont82 — DISMISSED** [review]

**Review by ammont82 — APPROVED** [review]

**Pull request merged**